src/Formplot/FileFormat/FormplotHelper.cs

```diff
@@ -93,9 +93,12 @@ private static bool IsTruncationSafe( byte[] content )
 					&& ( content[ 4 ] == 0x6C );
 		}
 
-		private static int FindEndOfContent( byte[] content )
+		private static int FindEndOfContent( byte[] content, int position )
 		{
-			var i = content.Length - 1;
+			if( position < 0 || position > content.Length - 1 )
+			 	return content.Length - 1;
+
+			var i = position;
 			while( i > -1 && content[ i ] == 0 )
```

Comment:
What do the zeros mean? Is the end of the content array filled up with 0?

Reply:
This code wasn't touched by me. I think, it's even your code.

Reply:
It's not. But it's fine then.